docs/src/poll.rst

```diff
@@ -108,6 +112,7 @@ API
         on the `events` field in the callback.
 
     .. versionchanged:: 1.9.0 Added the UV_DISCONNECT event.
+		.. versionchanged:: 1.10.0 Added the UV_PRIORITIZED event.
```

Comment:
s/1.10.0/1.11.0. The `1.10.0` is about to be released and this change will have to make it to the next version (I think).
Indentation is off